src/board/audio/sound-emoji-synthesizer.ts

```diff
@@ -0,0 +1,253 @@
+/**
```

Comment:
This is just the MakeCode code tweaked to compile here and use ESM.

Their TypeScript is older so I had fun with null/undefined fields so there's a bit more change than you might expect.